File: 方子樽/2021.01.28/ExcelSheetColumnNumber.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
const titleToNumber = function (columnTitle) {
    let ans = 0;
    for (let i = 0; i < columnTitle.length; i++) {
        //26進制
        ans = ans * 26 + columnTitle[i].charCodeAt() - 64;
    }
    return ans
};

File: 許靜玟/2022.01.28/ExcelSheetColumnNumber.cpp
class Solution {
public:
    int titleToNumber(string colTitle) {
        int ans = 0;
        for(int i=0; i< colTitle.length(); i++){
                if(i!=colTitle.length()-1) 
                    ans += (colTitle[i]-64)*pow(26, colTitle.length()-i-1);
                else 
                    ans += colTitle[i]-64;
        }
        return ans;
    }
};